src/core/utils/bin-sorter.js

```diff
@@ -79,7 +79,9 @@ export default class BinSorter {
    * @return {Number | Boolean} max count
    */
   getMaxCount() {
-    return Math.max(...this.sortedBins.map(b => b.counts));
+    let maxCount = 0;
+    this.sortedBins.map(x => (maxCount = maxCount > x ? maxCount : x));
```

Comment:
`map` creates and returns a new array of the same size. Better use `for of` or  possibly `forEach`.